semgrep_runner_v1.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package semgrep_runner_v1;
+
+option go_package = "semgrep_runner.v1";
```

Comment:
We use Go? There will be some Go client using the semgrep runner?

Reply:
This is an option for any clients in the future that _may_ use a Go client. It will ensure that the generated Go client has the proper package/namespace.

Reply:
Looks ugly to have this very Go specific thing in an interface file. There is no such thing for OCaml or python or typescript ...